### PR TITLE
gramatical fixes

### PR DESCRIPTION
**Pull request by slacle**
- added "a" on line 12
- changed "is" to "are"
- changed comma on line 15

### CODE REVIEW

File: _securityAdvice/040_passwords_en.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ There exist plenty of recommendations for secure passwords. Some of these are:
 
 - A password should not contain public or personal information like name of your pet, date of birth, or username.
 - A password should be long.
-- A password should not be an existing word or a combination of few words. It should be a combination of characters or words that is as random as possible.
+- A password should not be an existing word or a combination of a few words. It should be a combination of characters or words that are as random as possible.
```
[the remark this answers is not in the document]
Comment (overheadhunter) [reply _securityAdvice/040_passwords_en.md]: > That it refers to "combination" (singular).

jup

Comment (overheadhunter) [inline _securityAdvice/040_passwords_en.md]:
```suggestion
- A password should not be an existing word or a combination of a few words. It should be a combination of characters or words that is as random as possible.
```

[the remark this answers is not in the document]
Comment (slacle) [reply _securityAdvice/040_passwords_en.md]: I guess could be. To me it sounds more like it refers to "characters or words" ;)

[the remark this answers is not in the document]
Comment (overheadhunter) [reply _securityAdvice/040_passwords_en.md]: Maybe we can rephrase the sentence then? It should refer to the combination. While "correct horse battery staple" is a random combination of words, each of the words doesn't seem random to me. Can a word even be random? 🤔

[the remark this answers is not in the document]
Comment (slacle) [reply _securityAdvice/040_passwords_en.md]: I think it is ok with the "is" as you had it originally. At least grammatically I think it is correct. Maybe the best thing would be to give an example. 

And actually I think that the previous sentence was also grammatically correct as you had it originally. "a combination of few words" with or without the "a" should be both good. 

Another thing I noticed now is on line 10 where it says "information like name of your pet", should there be a "the" there? "information like the name of your pet"?

Really it was just the comma on line 15 that was really obvious that shouldn't be there that I initially wanted to change. And then I started to look if there was anything else that needed to change while I was there, but maybe I over analyzed things ;p

[the remark this answers is not in the document]
Comment (tobihagemann) [reply _securityAdvice/040_passwords_en.md]: You're right, I guess we didn't really proofread this section. Did a full review in a new commit: d9cdf0d8a630058450ba9a91e74cbf95e5cdb49b

**Pull request merged**